CHANGES.md

```diff
@@ -125,6 +125,14 @@ after the corresponding tracker issues.
   override this choice.
   <http://issues.fast-downward.org/issue1031>
 
+- For users: new LimitedPruning class replaces previous limitation
+  options of individual pruning methods
+  <http://issues.fast-downward.org/issue1042>
+  A previous command line option using this feature, such as
+  --search "astar(lmcut(),pruning=atom_centric_stubborn_sets(min_required_pruning_ratio=0.2,expansions_before_checking_pruning_ratio=1000))"
+  is now changed to
+  --search "astar(lmcut(),pruning=limited_pruning(pruning=atom_centric_stubborn_sets,min_required_pruning_ratio=0.2,expansions_before_checking_pruning_ratio=1000))
+
```

Comment:
I think this should be `pruning=atom_centric_stubborn_sets()`. The fact that the parentheses are currently optional is an option parser bug and shouldn't work anymore after the option parser rewrite. (Unless those involved in the rewrite thought it was a feature and explicitly added support. ;-) But I think we shouldn't.) I wouldn't write "for users"; see https://issues.fast-downward.org/msg10563.